dev-util/gammaray/gammaray-2.9.0.ebuild

```diff
@@ -0,0 +1,93 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit cmake-utils kde5-functions
+
+DESCRIPTION="High-level runtime introspection tool for Qt applications"
+HOMEPAGE="https://github.com/KDAB/GammaRay"
+
+if [[ ${PV} == 9999 ]] ; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/KDAB/GammaRay.git"
+else
+	SRC_URI="https://github.com/KDAB/GammaRay/releases/download/v${PV}/${P}.tar.gz"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-2 BSD MIT"
```

Comment:
README says it's GPL-2+.

How are the other licenses used? I see that they come from some bundled libraries. Is your ebuild suffering from them?

Reply:
I will take a look at this in a few days.

Reply:
@mgorny 
>README says it's GPL-2+.

I really can't remember why asturm proposed to include these too.

Reply:
@a17r ?

Reply:
I didn't look at the code myself, iirc you found those bundled libs in use were having these other licences.

Reply:
@mgorny @a17r 

Should we leave the LICENSE as is?

Reply:
I reread my own question now. I'm sorry that I was unclear. I was asking if the bundled libraries are actually being used, and what's the chance of using system libraries instead.

Reply:
@mgorny No worries. I am afraid I can't check this until Monday, but AFAIK `add_subdirectory(3rdparty/*` doesn't contain lz4 and the other directories can be disabled*.

*
```
    sed -i "/BackwardMacros.cmake/d" CMakeLists.txt || die
    sed -i "/add_backward(gammaray_core)/d" core/CMakeLists.txt || die
```
and 
```
		-DGAMMARAY_BUILD_UI=ON
		-DGAMMARAY_DISABLE_FEEDBACK=ON
```
are doing this thing.

Reply:
Sure, I don't think we are in a hurry here, are we?